admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/service/FormBuilderServiceImpl.java

```diff
@@ -731,6 +731,7 @@ public ListGrid buildCollectionListGrid(String containingEntityId, DynamicResult
     }
 
     protected boolean propertyExistsInResultSet(Property property, DynamicResultSet drs) {
+        if(property == null) return false;
```

Comment:
I think we prefer another code style
if (property == null) {
  return false;
}